docs/react-hook-form.md

```diff
@@ -1,10 +1,10 @@
-## ⚡ Forms handling
+# ⚡ Forms handling
 
```

Comment:
Maybe we should update this with latest chapter decision:
- only use translated error messages in schema
- use a custom resolver that handle defaults error messages (already implemented in hublo-front)

Reply:
I will update the docs once I'm done with translations (will be in another PR)